cirq/ops/controlled_gate_test.py

```diff
@@ -219,6 +219,11 @@ def test_reversible():
     assert (cirq.inverse(cirq.ControlledGate(cirq.S)) ==
             cirq.ControlledGate(cirq.S**-1))
 
+def test_phase_by():
```

Comment:
Add a test case where the underlying gate changes (e.g. use `cirq.google.ExpW`).

Reply:
Done